### ISSUE TITLE
Does not compile with dartdevc with implicit-casts: false

### ISSUE THREAD

**Issue by alextekartik** [dart-lang/logging#39]
I have a project (https://github.com/tekartik/sembast.dart) that does not compile with dartdevc if implicit-casts: false is set (in my project!)

```
Error compiling dartdevc module:logging|lib/lib__logging.js

[error] The argument type 'dynamic' can't be assigned to the parameter type 'String'. (package:logging/logging.dart, line 186, col 21)

Please fix all errors before compiling (warnings are okay).
```

The failing test: https://travis-ci.org/tekartik/sembast.dart/jobs/347553963

If I remove implicit-casts in my project, it compiles fine. Strong mode support needed?

**Comment by alextekartik** [comment]
It seems this package has not been published since 2016! although i had many updates since

**Comment by pq** [comment]
This should be fixed in `0.11.3+2`?  Please re-open if not!

**Comment by pq** [comment]
Great.  Thanks for following up!
